classroom.rb

```diff
@@ -0,0 +1,18 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def students
+    Student.all.select do |student|
+      student.classroom == self
+    end
+  end
+
+  def add_student(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] You need to push the student to the student variable you initialized too. Something like: `@student << student`

Reply:
* As mentioned by the previous CR , Kindly work on this HINT 💭 see the example below 👇 [visit this link for more ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)
```
class Owner
  ...

  def add_animal(animal)
    @animals.push(animal)
    animal.owner = self
  end
end
```